selfdrive/controls/controlsd.py

```diff
@@ -375,13 +376,6 @@ def update_events(self, CS):
 
     # TODO: fix simulator
     if not SIMULATION or REPLAY:
-      # Not show in first 1 km to allow for driving out of garage. This event shows after 5 minutes
-      if not self.sm['liveLocationKalman'].gpsOK and self.sm['liveLocationKalman'].inputsOK and (self.distance_traveled > 1500):
-        self.events.add(EventName.noGps)
```

Comment:
we still want this alert

Reply:
Added it back, via timing check of gpsLocation socket